app/src/main/java/com/willowtree/vocable/CategoriesUseCase.kt

```diff
@@ -21,6 +21,17 @@ class CategoriesUseCase(
     private val phrasesUseCase: PhrasesUseCase
 ) : ICategoriesUseCase {
 
+    private val unsortedCategories = combine(
+        storedCategoriesRepository.getAllCategories(),
+        presetCategoriesRepository.getPresetCategories()
+    ) { storedCategories, presetCategories ->
+        storedCategories.map { it.asCategory() } + presetCategories
+    }.map {
+        it.sortedBy { category ->
+            category.sortOrder
+        }
+    }
+
```

Comment:
this doesn't look right- old code? the variable is unused and the behavior (sorted) doesn't match the name (unsorted)

Reply:
This was an experiment that leaked through, good catch!